common/src/event.rs

```diff
@@ -31,27 +31,27 @@ use scale_info::TypeInfo;
 /// Same as `gear_core::message::DispatchKind`,
 /// but with additional info about reply.
 #[derive(Debug, Encode, Decode, Clone, PartialEq, Eq, TypeInfo)]
-pub enum Entry {
+pub enum MessageEntry {
     /// Init entry point.
-    Init,
+    InitEntry,
     /// Handle entry point.
-    Handle,
+    HandleEntry,
     /// Handle reply entry point.
-    Reply(MessageId),
+    HandleReplyEntry(MessageId),
     /// System signal entry point.
-    Signal,
+    HandleSignalEntry,
```

Comment:
I'm not sure that this is a good way to add the postfix `Entry`, because It's already clear from the name of the event